policy/modules/admin/usbguard.te

```diff
@@ -0,0 +1,66 @@
+policy_module(usbguard, 1.0)
+
+## <desc>
+##	<p>
+##	Determine whether usbguard-daemon can modify its rules in
+##	/etc/usbguard.
+##	</p>
+## </desc>
+gen_tunable(usbguard_modify_rule_files, false)
+
+gen_require(`
+	type var_log_t;
+	type var_t;
+        type sysfs_t;
+        class dir { add_name create getattr open read remove_name rmdir search setattr write };
+        class file { create getattr map open read setattr unlink write };
+        class filesystem getattr;
+        class lnk_file { getattr read };
+        class netlink_kobject_uevent_socket { bind create read setopt };
+        class unix_stream_socket { accept listen };
+')
+
+type usbguard_t;
+type usbguard_daemon_exec_t;
+init_daemon_domain(usbguard_t, usbguard_daemon_exec_t)
+
+type usbguard_dbus_t;
+type usbguard_dbus_exec_t;
+init_daemon_domain(usbguard_dbus_t, usbguard_dbus_exec_t)
+
+files_read_etc_files(usbguard_t)
+type usbguard_etc_t;
+files_config_file(usbguard_etc_t)
+allow usbguard_t usbguard_etc_t:dir { getattr open read search };
+allow usbguard_t usbguard_etc_t:file { getattr open read };
+
+tunable_policy(`usbguard_modify_rule_files',`
+	gen_require(`
+	        class dir { add_name create remove_name setattr write };
+		class file { create setattr unlink write };
+		')
+	allow usbguard_t usbguard_etc_t:dir { add_name create remove_name setattr write };
+	allow usbguard_t usbguard_etc_t:file { create setattr unlink write };
+')
+
+allow usbguard_t var_t:dir search;
+allow usbguard_t var_log_t:dir search;
+
+type usbguard_var_log_t;
+allow usbguard_t usbguard_var_log_t:file manage_file_perms;
+logging_log_file(usbguard_var_log_t)
+logging_log_filetrans(usbguard_t, usbguard_var_log_t, file)
+manage_files_pattern(usbguard_t, usbguard_var_log_t, usbguard_var_log_t)
+
+allow usbguard_t self:netlink_kobject_uevent_socket { bind create read setopt };
+allow usbguard_t self:unix_stream_socket { accept listen };
```

Comment:
If possible use object permissions set as defined in `policy/support/obj_perm_sets.spt`

Reply:
fixed